campaignion_layout/src/Context/LayoutCondition.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\campaignion_layout\Context;
+
+use Drupal\little_helpers\Services\Container;
+use Drupal\campaignion_layout\Lookup;
+
+/**
+ * Expose available layouts as a context condition.
+ */
+class LayoutCondition extends \context_condition {
+
+  /**
+   * Condition values.
+   */
+  public function condition_values() {
+    $values = [];
+    $themes = Container::get()->loadService('campaignion_layout.themes')->enabledThemes();
+    foreach ($themes as $theme) {
+      $values += $theme->layoutOptions(TRUE);
+    }
+    return $values;
```

Comment:
This can be replaced now by `return Container::get()->loadService('campaignion_layout.themes')->layoutOptions();`.